api/models/templates/service/redis.tmpl

```diff
@@ -2,6 +2,11 @@
   {
     "AWSTemplateFormatVersion" : "2010-09-09",
     "Parameters": {
+      "AutomaticFailoverEnabled": {
+        "Type": "String",
```

Comment:
Sure thing - first example of a bool I came across was https://github.com/convox/rack/blob/master/api/models/templates/service/mysql.tmpl#L20-L23, so I just followed the same pattern.. will update.